src/content/docs/design/content-customization/update-auth-page-content.mdx

```diff
@@ -0,0 +1,70 @@
+---
+page_id: a643d22e-c26c-4c14-9b95-76aea37352ef
+title: Update page content in the auth flow
+sidebar:
+  order: 2
+relatedArticles:
+  - b3a309c6-af3c-4155-b39e-7e1f6e83721f
+  - 2aa551b8-06c0-4947-bd4b-d643c77ed224
+  - d5ddfbe2-6608-45d4-8c46-44be93690c9d
+
+---
+
+For maximum extensibility, page content has been decoupled from the page itself. 
+
+This lets you update the copy on most screens, which you can do via the **Page content** section of the admin area (see below) or via API (coming soon).
+
+### Update page content
+
+If your application is build for [multiple languages](https://docs.kinde.com/design/pages/internationalization), select these first in Kinde. 
+
+1. In Kinde, go to **Design > Page content**.
+2. Select the language you want (only applicable if multiple languages are configured).
+3. Select the page you want to edit. The list of content shown corresponds with sections as they appear on the page to the user. For example, Main heading, Email field, etc.
+4. Make the changes to the content. If you want, you can use text variables, explained below.
+5. When you’ve finished, select **Save**. 
+6. Check changes by selecting **Preview**.
+
+### Text variables for page content
+
+Variables are used to stand in for actual values that you want to appear on pages. They are a way of automating content. For example, if you use the `${email_address}` variable, the user’s email address will be shown.
+
+Variables can be used on pages as follows. 
+
+**Sign in page**
+
+`${social_provider_name}` // for use on the social buttons
+
+`${enterprise_provider_name}` // for use on the enterprise SSO buttons
+
+**Sign up page**
+
+`${social_provider_name}` // for use on the social buttons
+
+`${enterprise_provider_name}` // for use on the enterprise SSO buttons
+
+**Verify password**
```

Comment:
@clairekinde11 should we axe the ones you can't edit the fields with those relevant placeholders yet. i.e remove
* Verify password
* SMS verify code page
* Sign in page
* Sign up page